odoo/tools/config.py

```diff
@@ -335,6 +332,19 @@ def _build_cli(self):
                          help="Absolute path to the GeoIP Country database file.")
         parser.add_option_group(group)
 
+        group = optparse.OptionGroup(parser, "Shell options")
```

Comment:
Just a generic question: Cannot this group be added only when executing `odoo.cli.shell`?
Like in neutralize or populate scripts, this is an option just for that cli.

@Julien00859 What do you think?

Reply:
Why? Newbie question, can't we go and re-read a section of the file?

Reply:
In my other work I emit a warning when you use another section than `[options]`, I also emit a warning for any unknown option. So if you run the `server` subcommand you would have a warning because `shell_interface` wouldn't be recognized (annoying).

Reply:
and btw the `server` subcommand is always ran when the first argv starts in `--addons-path=`, so even when using the `shell` subcommand it would emit a (silly) warning the first time the config is loaded

Reply:
@Julien00859 After you merge the refactoring PR, maybe we can split the options of server from the global ones. Maybe we should allow for other sections than options.